go/worker/common/p2p/init.go

```diff
@@ -37,6 +43,8 @@ func init() {
 	Flags.StringSlice(cfgP2pAddresses, []string{}, "Address/port(s) to use for P2P connections when registering this node (if not set, all non-loopback local interfaces will be used)")
 	Flags.Int64(CfgP2PPeerOutboundQueueSize, 32, "Set libp2p gossipsub buffer size for outbound messages")
 	Flags.Int64(CfgP2PValidateQueueSize, 32, "Set libp2p gossipsub buffer size of the validate queue")
+	Flags.Int64(CfgP2PValidateConcurency, 1024, "Set libp2p gossipsub per topic validator concurency limit")
+	Flags.Int64(CfgP2PValidateThrottle, 8192, "Set libp2p gossipsub validator concurency limit")
```

Comment:
Are these defaults that existed before?

Reply:
These are the defaults yes https://github.com/libp2p/go-libp2p-pubsub/blob/e6ad80cf4782fca31f46e3a8ba8d1a450d562f49/validation.go#L14-L16